### PR TITLE
Page favorites

### PR DESCRIPTION
**Pull request by OlegNelasy**
made a favorites page

### CODE REVIEW

File: src/components/Header/Header.module.scss [hunk 1]
```diff
@@ -33,19 +33,34 @@ $button-color: rgba(179,178,188,1);
...
+        //     flex-direction: column;
+            
+        //     .button {
+        //         border-radius: 3px;
+        //         border: 1px solid $button-color;
+        //         background-color: $background-color;
+        //         color: $button-color;
+        //         padding: 5px;
+        //     }
+        // }
```
Comment (AOnTeb9Celuet) [inline src/components/Header/Header.module.scss]: remove comments

File: src/components/Header/index.jsx [hunk 2]
```diff
@@ -11,11 +11,14 @@ export default function Header() {
           <img className={styles.img} src={logo} alt="logo" />
           <h1 className={styles.movies}>Movies</h1>
         </NavLink>
-        <div className={styles.boxButton}>
+        {/* <div className={styles.boxButton}>
           <button className={styles.button} name="myAccount">
             My Account
           </button>
-        </div>
+        </div> */}
```
Comment (AOnTeb9Celuet) [inline src/components/Header/index.jsx]: remove comments

File: src/pages/Favorites/Favorites.module.scss [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            
+            .unFavoriteButton {
+                position: absolute;
+                top: 0;
+                right: 0;        
+                text-decoration: none;
+                color: $darck-color;
+                font-size: 40px;
+                font-weight: 400; 
+                border: 2px solid $darck-color;
```
Comment (AOnTeb9Celuet) [inline src/pages/Favorites/Favorites.module.scss]:
```suggestion
                border: 2px solid $dark-color;
```

File: src/pages/Favorites/Favorites.jsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import FavoriteMovie from "./FavoriteMovie";
+import { useEffect, useState } from "react";
+import { FAVORITE_MOVIES } from "../../constants/constants.js";
+
+import styles from "./Favorites.module.scss";
+
+export const Favorites = () => {
+  const [favoriteMovies, setFavoriteMovies] = useState([]);
+
+  const getFavoriteMovies = async () => {
```
Comment (AOnTeb9Celuet) [inline src/pages/Favorites/Favorites.jsx]: u dont need async right here cause this function doesnt have an async actions inside. Actually, u dont even need this function, just move the 11-12 lines code to the useEffect directly

File: src/pages/Favorites/FavoriteMovie.jsx [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+            alt={movie.title}
+          />
+        </div>
+        <div className={styles.movieInformation}>
+          <p className={styles.movieTitle}>{movie.title}</p>
+          <p className={styles.movieOverview}>{movie.overview}</p>
+        </div>
+      </NavLink>
+      <button
+        className={styles.unFavoriteButton}
```
Comment (AOnTeb9Celuet) [inline src/pages/Favorites/FavoriteMovie.jsx]:
```suggestion
        className={styles.unfavoriteButton}
```

File: src/pages/Favorites/Favorites.module.scss [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+                    .movieOverview {
+                        margin-top: 50px;
+                        font-weight: 400;
+                        color: $darck-color;
+                        font-size: 30px;
+                    }
+                }
+            }
+            
+            .unFavoriteButton {
```
Comment (AOnTeb9Celuet) [inline src/pages/Favorites/Favorites.module.scss]:
```suggestion
            .unfavoriteButton {
```

**Review by AOnTeb9Celuet — CHANGES_REQUESTED** [review]

**Comment by OlegNelasy** [comment]
Made corrections in a new commit

**Comment by OlegNelasy** [comment]
Made corrections in a new commit

**Review by AOnTeb9Celuet — APPROVED** [review]

**Pull request merged**